examples/Backtesting/Backtesting.md

```diff
@@ -0,0 +1,17 @@
+# Backtesting
+
+These examples demonstrate the backtesting capabilities of BayBE.
+By "backtesting", we refer to the process of setting up a campaign, getting recommendations
+and adding measurements in an iterative fashion.
```

Comment:
suggest to add `.... in an iterative fashion by using an existing lookup in the form of historical data or analytical functions.`

Reply:
Included.